android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java

```diff
@@ -0,0 +1,269 @@
+        mReactRootView.startReactApplication(mReactInstanceManager, "App", props);
+        mReactRootView.setBackgroundColor(BACKGROUND_COLOR);
+        addView(mReactRootView);
+    }
+
+    /**
+     * Setter for the {@JitsiMeetView.Listener} set on this view.
+     *
+     * @param listener - Listener for this view.
+     */
+    public void setListener(Listener listener) {
+        this.listener = listener;
+    }
+
+    /**
+     * Activity lifecycle method which should be called from <tt>Activity.onBackPressed</tt> so
+     * we can do the required internal processing.
+     *
+     * @return - true if the back-press was processed, false otherwise. In case false is returned
+     * the application should call the parent's implementation.
+     */
+    public static boolean onBackPressed() {
+        if (mReactInstanceManager != null) {
+            mReactInstanceManager.onBackPressed();
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * Activity lifecycle method which should be called from <tt>Activity.onDestroy</tt> so
+     * we can do the required internal processing.
+     *
+     * @param activity - <tt>Activity</tt> being destroyed.
+     */
+    public static void onHostDestroy(Activity activity) {
+        if (mReactInstanceManager != null) {
+            mReactInstanceManager.onHostDestroy(activity);
+        }
+    }
+
+    /**
+     * Activity lifecycle method which should be called from <tt>Activity.onPause</tt> so
+     * we can do the required internal processing.
+     *
+     * @param activity - <tt>Activity</tt> being paused.
+     */
+    public static void onHostPause(Activity activity) {
+        if (mReactInstanceManager != null) {
+            mReactInstanceManager.onHostPause(activity);
+        }
+    }
+
+    /**
+     * Activity lifecycle method which should be called from <tt>Activity.onResume</tt> so
+     * we can do the required internal processing.
+     *
+     * @param activity - <tt>Activity</tt> being resumed.
+     */
+    public static void onHostResume(Activity activity) {
+        if (mReactInstanceManager != null) {
+            mReactInstanceManager.onHostResume(activity, null);
+        }
+    }
+
+    /**
+     * Activity lifecycle method which should be called from <tt>Activity.onNewIntent</tt> so
+     * we can do the required internal processing. Note that this is only needed if the activity's
+     * "launchMode" was set to "singleTask". This is required for deep linking to work once the
+     * application is already running.
+     *
+     * @param intent - <tt>Intent</tt> instance which was received.
+     */
+    public static void onNewIntent(Intent intent) {
+        if (mReactInstanceManager != null) {
+            mReactInstanceManager.onNewIntent(intent);
+        }
+    }
+
+    /**
+     * Interface for listening to events coming from Jitsi Meet.
+     */
+    public interface Listener {
```

Comment:
If all methods raise `UnsupportedOperationException`, then the implementers are better off without it (because they'll at least suffer the performance penalty incurred by throwing exceptions). If all methods are empty/do nothing, then it's a help because the implementers can override whichever they are interested in. Additionally, "empty methods" is what you gave iOS developers.